### PR TITLE
fix(docs): added max-width to text & images

### LINKED ISSUE #1133: Add max-width to markdown content
Add a max-width to prevent content from pages authored in markdown from stretching across the entire width of a page. 

Pages impacted:
All content in getting started section
Design guidelines > Usage and behavior
Design guidelines > Content
All descriptions at the top of the component docs pages

### PR DESCRIPTION
**Pull request by evwilkin**
resolves #1133

### CODE REVIEW

File: packages/patternfly-3/Gemfile.lock [hunk 1]
```diff
@@ -47,6 +47,3 @@ DEPENDENCIES
   jekyll-sitemap
   json
   rack
-
-BUNDLED WITH
```
Comment (redallen) [inline packages/patternfly-3/Gemfile.lock]: Removing the BUNDLED WITH worked! Nice!!!

**Comment by evwilkin** [comment]
> Core preview: https://patternfly-org-pr-1616-next.surge.sh
> React preview: https://patternfly-org-pr-1616-react.surge.sh
> Org preview: https://patternfly-org-pr-1616-org.surge.sh

@rachael-phillips @gdoyle1 looks like this got merged already, but visual review of the changes would still be good!

**Comment by gdoyle1** [comment]
@evwilkin I checked it out yesterday and it looked good!

**Comment by rachael-phillips** [comment]
@gdoyle1 @mcarrano @evwilkin  I really like the width changes - a great improvement! The only thing that looks a bit strange to me is that the tables are half the length of our examples on the component pages. As a user, I'm not sure I would make the connection as to why the styling is different. Here's a view from where the example differs from the table: 

Screen Shot 2020-01-15 at 9 15 51 AM

What do you think about the idea of all of our tables still being full width? Curious to hear your thoughts!

**Comment by gdoyle1** [comment]
> What do you think about the idea of all of our tables still being full width? Curious to hear your thoughts!

@rachael-phillips @evwilkin @mcarrano I agree that on the components pages, we should expand our table width to match the component width! I don't think we should do that EVERYWHERE though - for example, on the typography page, the width it is at now looks good to me.
Screen Shot 2020-01-15 at 9 46 36 AM

**Comment by evwilkin** [comment]
I agree - picking up this change in #1622

**Comment by rachael-phillips** [comment]
Thanks @gdoyle1 and @evwilkin !

**Pull request merged**